scripts/helper/kvm/virsh-cleanup.sh

```diff
@@ -22,6 +22,12 @@ declare -i nvols
 pools=$(virsh pool-list --all | grep test-ocp | awk '{print $1}')
 for i in $pools
 do
+	pool_status=$(virsh pool-info $i | grep State | awk '{ print $2 }')
+	if [[ "$pool_status" == "inactive" ]]; then
+		echo "virsh pool-undefine $i"
+		virsh pool-undefine $i
```

Comment:
At the end of the function, you will see a second virsh pool-undefine $i.   Do we need to add a continue stmt to avoid the rest of the code in the loop?   Better yet, can't we add an if [ poolstatus != inactive ] to skip the instructions that don't apply.

Reply:
@lukebrowning please review the code changes as per you comment